# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/security/test/unit/xpcshell.toml
# Commit: f3b4d635577f
# Full Hash: f3b4d635577f5bfa3fc51024f96f58b2d3019912
# Author: Frédéric Wang <fwang@igalia.com>
# Date: 2025-01-23 21:50:52
# Description:
#   Bug 1942517 - Skip trusted type check for DOMParser when global object is not a Window. r=smaug
#   
#   Adds tests for TT sinks when the global object is neither a Window nor a
#   WorkerGlobalScope and adjust TrustedTypeUtils accordingly.
#   
# ==============================================================================

diff -r c4c21220ccc5 -r f3b4d635577f dom/security/test/unit/xpcshell.toml
--- a/dom/security/test/unit/xpcshell.toml	Thu Jan 23 13:32:15 2025 +0000
+++ b/dom/security/test/unit/xpcshell.toml	Thu Jan 23 13:44:09 2025 +0000
@@ -12,3 +12,5 @@
 ["test_https_only_https_first_prefs.js"]
 
 ["test_isOriginPotentiallyTrustworthy.js"]
+
+["test_get_trusted_types_compliant_string.js"]